plugins/services/src/js/components/forms/VolumesFormSection.js

```diff
@@ -242,6 +243,38 @@ class VolumesFormSection extends Component {
         .at(key + offset, {})
         .get(this.props.errors).containerPath;
 
+      const dockerType = findNestedPropertyInObject(
+        this.props.data,
+        "container.type"
+      );
+
+      let sizeField = (
+        <Tooltip
+          content="Docker Runtime only supports the default size for implicit volumes, please select Mesos Runtime if you want to modify the size."
```

Comment:
Well since it is in the create service modal which is not below the provider in the react tree it is not so simple. We should fix this in another PR.